linux_os/guide/services/ssh/ssh_server/sshd_use_approved_ciphers/rule.yml

```diff
@@ -60,7 +60,7 @@ references:
     cui: 3.1.13,3.13.11,3.13.8
     disa: 68,366,803
     hipaa: 164.308(b)(1),164.308(b)(2),164.312(e)(1),164.312(e)(2)(i),164.312(e)(2)(ii),164.314(b)(2)(i)
-    nist: AC-3,AC-17(a),AC-17(b),AC-17(2),AU-10(5),CM-6(b),IA-5(1)(c),IA-7,SI-7,SC-13
+    nist: AC-3,AC-17(a),AC-17(b),AC-17(2),AU-10(5),CM-6(b),IA-5(1)(c),IA-7,MA-4(6),SI-7,SC-13,SC-12(3)
```

Comment:
Should SC-12(2) be added?


SC-12(2) | CRYPTOGRAPHIC KEY ESTABLISHMENT AND MANAGEMENT \| SYMMETRIC KEYSThe organization produces, controls, and distributes symmetric cryptographic keys using [Selection: NIST FIPS-compliant; NSA-approved] key management technology and processes.
-- | --